adapters/devices/sofar/lsw.go

```diff
@@ -73,64 +77,34 @@ func (l LSWRequest) checksum(buf []byte) uint8 {
 	return checksum
 }
 
+var AllRegisterRanges = []registerRange{
```

Comment:
Rather than work in an interface for a struct in the `sofar` package that the logger struct will need to implement (and then subsequently pass the logger instance to the metrics package), I've implemented an exported non-typed method `GetAllRegisterNames` to just return the full list of register names. Since the data is static, I thought this was a better fit.